### PR TITLE
[huggingface] Add build and test workflows

### PR DESCRIPTION
**Pull request by xyang16**
*Issue #, if available:*

*Description of changes:* Add build and test workflows

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .github/workflows/build-huggingface.yml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+jobs:
+  create-runner:
+    runs-on: [ self-hosted, scheduler ]
+    steps:
+      - name: Create new CPU instance
+        id: create_cpu
+        run: |
+          cd /home/ubuntu/djl_benchmark_script/scripts
+          token=$( curl -X POST -H "Authorization: token ${{ secrets.ACTION_RUNNER_PERSONAL_TOKEN }}" \
+          https://api.github.com/repos/deepjavalibrary/djl-serving/actions/runners/registration-token \
```
Comment (lanking520) [inline .github/workflows/build-huggingface.yml:32]: repo needs to change, you also need to register a personal token

File: .github/workflows/build-huggingface.yml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+          --fail \
+          | jq '.token' | tr -d '"' )
+          ./start_instance.sh action_cpu $token djl-serving
```
Comment (lanking520) [inline .github/workflows/build-huggingface.yml:35]: rename to your own project

File: .github/workflows/build-huggingface.yml [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        with:
+          registries: "125045733377"
+      - name: Clean docker env
+        run: |
+          yes | docker system prune -a --volumes
+      - name: Build and push docker image
+        uses: docker/build-push-action@v4
+        env:
+          REGISTRY: ${{ steps.login-ecr.outputs.registry }}
+          REPOSITORY: djl-serving
```
Comment (lanking520) [inline .github/workflows/build-huggingface.yml:78]: need to change

File: .github/workflows/test-huggingface.yml [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+jobs:
+  create-runners:
+    runs-on: [self-hosted, scheduler]
+    steps:
+      - name: Create new G5 instance
+        id: create_gpu
+        run: |
+          cd /home/ubuntu/djl_benchmark_script/scripts
+          token=$( curl -X POST -H "Authorization: token ${{ secrets.ACTION_RUNNER_PERSONAL_TOKEN }}" \
+          https://api.github.com/repos/deepjavalibrary/djl-serving/actions/runners/registration-token \
```
Comment (lanking520) [inline .github/workflows/test-huggingface.yml:32]: same applies to here

**Review by lanking520 — CHANGES_REQUESTED** [review]

File: .github/workflows/test-huggingface.yml (hunk 4 above)
Comment (xyang16) [reply .github/workflows/test-huggingface.yml]: Done

**Review by lanking520 — APPROVED** [review]

**Pull request merged**